actors/builtin/miner/miner_actor.go

```diff
@@ -1139,7 +1139,7 @@ func (a Actor) DeclareFaultsRecovered(rt Runtime, params *DeclareFaultsRecovered
 
 type CompactPartitionsParams struct {
 	Deadline   uint64
-	Partitions *abi.BitField
+	Partitions bitfield.BitField
```

Comment:
note: I switched from `abi.BitField` to `bitfield.BitField` to be consistent. We can also go the other way if we want to alias everything inside `abi`.

Reply:
The name stuttering is unfortunate, but generally 👍 ; aliasing was an interim thing when the bitfield package didn't have everything we needed.